### DISCUSSION TITLE
RTF 1.2.0 Released

### DISCUSSION THREAD

**Discussion by drdanz** [robotology/community#234] [Releases]
RTF 1.2.0 was released today and is now available for download at: https://github.com/robotology/robot-testing/releases/

## Important Changes

* The YARP middleware plugin is no longer in RTF, but it was moved in YARP.

**Comment by apaikan** [comment]
have the following repos also been updated? 
- iCub tests
- VVV school assignments

**Comment by pattacini** [comment]
vvv-school assignments were already aligned with the changes.

**Comment by drdanz** [comment]
@apaikan icub-tests devel was merged into master and it was tagged together with icub-main

**Comment by apaikan** [comment]
thumbs up guys!  
@drdanz : please go ahead to close this! 
thanks :)
